### DISCUSSION TITLE
Social Media Icons

### DISCUSSION THREAD

**Discussion by saadusufzai** [tailwindlabs/heroicons#596] [Ideas]
Kindly add social media icons or let us know why you guys are not adding them in the library. 
I created an issue 5 month ago and many other people also created the issue . but the issues are closed without any explanations. 
let us know so that we move to some other library or create our own

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
